components/Statistics/TourStats.tsx

```diff
@@ -41,39 +33,23 @@ export default function TourStats(props: TourStatsProps) {
 
   return (
     <View style={styles.container}>
-      <View style={styles.progressContainer}>
-        <ProgressBar progress={progress} style={styles.progressBar} />
-        <Text
-          //@ts-ignore
-          style={{
-            position: "absolute",
-            top: "28%",
-            left:
-              progress < 0.18
-                ? `${((progress + 0.04) * 100).toFixed(2)}%`
-                : progress < 1
-                  ? `${((progress - 0.125) * 100).toFixed(2)}%`
-                  : "44%",
-            color: progress >= 0.15 ? "#fff" : theme["text-basic-color"],
-            fontSize: 19,
-          }}
-        >
-          {(progress * 100).toFixed(1) + "%"}
-        </Text>
-      </View>
+      {
+        // Fortschrittsbalken über die Tourdistanz
+        <TourProgressBar progress={progress} style={styles.progressContainer} />
+      }
       <View style={styles.statsContainer}>
         <View style={styles.stat_column}>
           <View style={styles.stat_row}>
             <Icon name="arrows-left-right" style={styles.icon_style} />
-            <Text>{getTourDistance(stages).toFixed(1) + " km"}</Text>
+            <Text>{getTourDistanceString(stages)}</Text>
           </View>
           <View style={styles.stat_row}>
             <Icon name="arrow-up-right-dots" style={styles.icon_style} />
             <Text>{"0 m"}</Text>
           </View>
           <View style={styles.stat_row}>
             <Icon name="gauge-high" style={styles.icon_style} />
-            <Text>{getTourAverageSpeed(stages).toFixed(1) + " km/h"}</Text>
+            <Text>{getTourAverageSpeedString(stages)}</Text>
```

Comment:
Könnte man hier auch über all deine Komponente benutzen?

Reply:
Bzw soll ich das dann in meinem Refactoring machen?

Reply:
Ja genau, dafür hab ich sie gemacht :) 

Man kann da auch alles mögliche anpassen (Icongröße, Schriftgröße, Iconfarbe)